docs/src/main/sphinx/connector/memory.md

```diff
@@ -25,6 +25,19 @@ CREATE TABLE example.default.nation AS
 SELECT * from tpch.tiny.nation;
 ```
 
+Create the table orders if it does not already exist, adding a table comment, adding a column comment and a column with default value:
```

Comment:
This description is too long, the scope is the `DEFAULT` functionality, please remove the “already exists” part